core/src/services/dropbox/builder.rs

```diff
@@ -114,6 +114,7 @@ impl Builder for DropboxBuilder {
 
     fn from_map(map: HashMap<String, String>) -> Self {
         let mut builder = Self::default();
+        builder.root(map.get("root").map(|v| v.as_str()).unwrap_or_default());
```

Comment:
How about keeping the same style that `map.get("root").map(|v| builder.root(v));`